server/middleware/authentication.js

```diff
@@ -364,3 +383,36 @@ export function mustBeLoggedIn(req, res, next) {
     }
   });
 }
+
+export const checkTwoFactorAuthJWT = (req, res, next) => {
+  const header = req.headers && req.headers.authorization;
+  if (!header) {
+    return next(new BadRequest('Requires Authorization headers.'));
+  }
+
+  const parts = header.split(' ');
+  const scheme = parts[0];
+  const token = parts[1];
+  if (!/^Bearer$/i.test(scheme) || !token) {
+    return next(new BadRequest('Format is Authorization: Bearer [token]'));
+  }
```

Comment:
Could you extract this in an helper and share it with the other function from where it was copy/pasted?